### DISCUSSION TITLE
How to stop Weblate from escaping Android XML too much

### DISCUSSION THREAD

**Discussion by TheLastProject** [WeblateOrg/weblate#5657] [Q&A] [answered]
Hey all,

Weblate escapes my Android XML too much, forcing me to unescape it myself manually. Is there anything I can do to prevent this?

See https://github.com/TheLastProject/Catima/issues/154 for more info and https://github.com/TheLastProject/Catima/pull/163 for the most recent PR I had to do this in.

**Comment by nijel** [comment, marked as answer]
It looks like a bug in the [translate-toolkit](https://github.com/translate/translate/) the well-formed XML should be stored unescaped.

**Reply by nijel** [reply]
Hmm, I've added tests in https://github.com/translate/translate/commit/a76873e019c8791a37b404682cbe122f36b256e9 and it seems to work fine there...

**Reply by TheLastProject** [reply]
Very odd... Because well, it is still happening for me today: https://github.com/TheLastProject/Catima/pull/174

Maybe should move this into a GitHub issue?

**Reply by nijel** [reply]
I've moved this to a pull request fixing this, see https://github.com/translate/translate/pull/4305 :smiley:. It was caused by a namespace defined on the `<resources>`.

**Reply by TheLastProject** [reply]
Awesome! I'll keep an eye on the automatic translation PRs and when I can confirm this issue is no longer happening I'll mark this "discussion" as resolved :)

**Reply by nijel** [reply]
I will probably deploy the fix today, but I can't promise that.

**Reply by TheLastProject** [reply]
I can confirm since https://github.com/TheLastProject/Catima/pull/192 that it no longer gets mangled. Awesome. Thank you!
